resources/lang/de/filter.json

```diff
@@ -0,0 +1,6 @@
+{
+  "Select option": "Auswählen",
+  "Press enter to select": "Enter, um auszuwählen",
```

Comment:
Maybe a better translation:
```suggestion
  "Press enter to select": "Enter drücken, um auszuwählen",
```

Reply:
I deliberately kept it short. There is not much space.